frontend/src/components/UserPage/OfferedServices/OfferedServices.tsx

```diff
@@ -53,42 +70,87 @@ export const OfferedServices: FunctionComponent = () => {
             body: JSON.stringify({
                 serviceId: +chosenService,
                 price: +price,
```

Comment:
'+' sign works ok to cast 'price' variable to number, but I think it is better to not use such language-specific syntax and keep up with more standard one such as 'price as number' or '<number>price'.

Reply:
`price as number` will not make a real cast but only override its ts type. We can cast price to number with `+price` or with `Number.parseFloat(price)`